### PR TITLE
Add SS gunicorn setup / fix MCPServer upstart

### PR DESCRIPTION
**Pull request by hakamine**
- Add tasks to configure SS to run using gunicorn instead of uwsgi (uwsgi is still the default)
- Fix MCPServer upstart script (it wasn't starting on reboot)

### CODE REVIEW

File: tasks/ss-websrv-gunicorn.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  template:
+    src: "etc_nginx_sites_storage-gunicorn.conf.j2"
+    dest: "/etc/nginx/sites-available/storage.conf"
+    backup: "yes"
+  tags: "amsrc-ss-websrv"
+
+- name: "set gunicorn init file"
+  template:
+    src: "etc_init_archivematica-ss.conf.j2"
+    dest: "/etc/init/archivematica-ss.conf"
```
Comment (sevein) [inline tasks/ss-websrv-gunicorn.yml]: `/etc/init/archivematica-storage-service`?

File: templates/etc_init_archivematica-ss.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+env SS_DB_HOST={{ archivematica_src_ss_env_ss_db_host }}
+env SS_DB_NAME={{ archivematica_src_ss_env_ss_db_name }}
+env SS_DB_PASSWORD={{ archivematica_src_ss_env_ss_db_password }}
+env SS_DB_USER={{ archivematica_src_ss_env_ss_db_user }}
+
+exec /usr/share/python/archivematica-storage-service/bin/gunicorn storage_service.wsgi:application  \
+--workers {{ archivematica_src_ss_gunicorn_workers }} \
+--timeout {{ archivematica_src_ss_gunicorn_timeout }} \
+--log-level error \
+--bind {{ archivematica_src_ss_gunicorn_bind }}
```
Comment (sevein) [inline templates/etc_init_archivematica-ss.conf.j2]: Can you add `--reload` when the development mode is enabled? https://github.com/artefactual-labs/ansible-role-archivematica-src/tree/dev/venvs-everywhere/templates/etc/archivematica

File: templates/etc_nginx_sites_storage-gunicorn.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    location / {
+    proxy_set_header Host $http_host;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_redirect off;
+    proxy_buffering off;
+
+    proxy_read_timeout {{ archivematica_src_ss_nginx_proxy_read_timeout }};
+    proxy_pass {{ archivematica_src_ss_nginx_proxy_pass }};
+
+    }
```
Comment (sevein) [inline templates/etc_nginx_sites_storage-gunicorn.conf.j2]: Indent?

**Comment by hakamine** [comment]
I fixed the service name to be archivematica-storage-service instead of archivematica-ss (at first I was wary of using a long name but it seems to be common practice with upstart service names )

File: templates/etc_init_archivematica-ss.conf.j2 [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# {{ ansible_managed }}
+description "Archivematica Storage Service"
+author          "Artefactual Systems"
```
Comment (sevein) [inline templates/etc_init_archivematica-ss.conf.j2]: Unnecessary white spaces?

**Comment by sevein** [comment]
👍

**Pull request merged**